tests/ui_tests/__init__.py

```diff
@@ -18,6 +18,7 @@
 SCREENS_DIR = UI_TESTS_DIR / "screens"
 HASH_FILE = UI_TESTS_DIR / "fixtures.json"
 SUGGESTION_FILE = UI_TESTS_DIR / "fixtures.suggestion.json"
+FIXTURES_DIFF = UI_TESTS_DIR / "fixtures.diff.json"
```

Comment:
looks like the file is not valid json but one-json-per-line right? in that case let's call it `fixtures.json.diff` so that the suffix matches

Reply:
Yes, they are individual JSON objects on each line.

Done like this to be "multicore-friendly", so that the file is append-only and there is no need for re-writing the whole file when adding a new record (which could cause some race-conditions when two processes/threads would do it simultaneously). 

Name changed in [c4d30db](https://github.com/trezor/trezor-firmware/pull/2744/commits/c4d30dbd1c5618f7dab668c584c819826c8dd230)